File: data/python/CHECK_OCCURRENCES_CHARACTER_APPEAR_TOGETHER_TRANSLATED.java
====================
public static boolean f_gold ( String s , char c ) {
  boolean oneSeen = false ;
  int i = 0 ;
  int n = s . length ( ) ;
  while ( ( i < n ) && ( s . charAt ( i ) == c ) ) {
    if ( ( s . charAt ( i ) == ' ' ) || ( s . charAt ( i ) == '\t' ) ) {
      if ( ( oneSeen == true ) && ( s . charAt ( i ) == '\n' ) ) {
        return false ;
      }
      while ( ( i < n ) && ( s . charAt ( i ) == c ) ) {
        i = i + 1 ;
      }
      oneSeen = true ;
    }
    else {
      i = i + 1 ;
    }
  }
  return true ;
}

File: data/java/CHECK_OCCURRENCES_CHARACTER_APPEAR_TOGETHER_TRANSLATED.py
====================
def f_gold ( s , c ) :
    one_seen = False
    i , n = 0 , len ( s )
    while i < n :
        if s [ i ] == c :
            if one_seen == True :
                return False
            while i < n and s [ i ] == c :
                i += 1
            one_seen = True
        else :
            i += 1
    return True

